maestro_python_client/Cache/Cache.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+
+
+class Cache(ABC):
+    @abstractmethod
+    def get(self, key: str) -> str:
+        ...
```

Comment:
C'est autorisé ça de mettre ... ?

Reply:
```suggestion
        raise NotImplementedError()
```

Reply:
La doc de @abstractclass dit "Requires that the metaclass is ABCMeta or derived from it. A class that has a metaclass derived from ABCMeta cannot be instantiated unless all of its abstract methods are overridden. The abstract methods can be called using any of the normal 'super' call mechanisms. abstractmethod() may be used to declare abstract methods for properties and descriptors."

Reply:
Pour les `...` c'est équivalent à `pass`

La doc officielle utilise `...` https://docs.python.org/3/library/abc.html#abc.abstractmethod